File: homework2/lines.js
""" Ronald Uy & Edwin Cheung 
	CMSI 386 - Homework #2 """

var sys = require('sys'),
	fs = require('fs');
var filename = process.argv[2];
fs.readFile(filename, 'utf8', function (read_error,content){
	var count = 0;
	content.split('\n').forEach(function(line){
		if(line.replace(/\s*\/\/.*/, "").length>0) count++;
	});
	sys.puts(count);
});

File: homework1/lines.py
"""
	Ronald Uy & Edwin Cheung
	CMSI 386 Homework 1 - Dr. Toal
"""

import sys
with open(sys.argv[1]) as f:
	count=0
	symbol=0
	for line in f:
		num_lines = sum(1 for line in f if line.strip('\n'))
	if line.strip('#'):
		symbol+=1
	if len(line)-len(line.strip())>0 :
		count+=1
	num_lines=num_lines-symbol-count
	print num_lines
f.close()
